logging/unit_tests/test__gax.py

```diff
@@ -412,7 +412,7 @@ def test_write_entries_w_extra_properties(self):
             'severity': SEVERITY,
             'labels': LABELS,
             'insertId': IID,
-            'timestamp': NOW,
+            'timestamp': NOW.strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
```

Comment:
This is added because this would be a string at this point but this test skips the further up building of the request.